opal/mca/btl/smcuda/btl_smcuda.c

```diff
@@ -887,8 +887,7 @@ int mca_btl_smcuda_sendi(struct mca_btl_base_module_t *btl,
         /* note that frag==NULL is equivalent to rc returning an error code */
         MCA_BTL_SMCUDA_FRAG_ALLOC_EAGER(frag);
         if (OPAL_UNLIKELY(NULL == frag)) {
-            *descriptor = NULL;
-            return OPAL_ERR_OUT_OF_RESOURCE;
+            goto return_resource_busy;
```

Comment:
Is it ok that this now returns `RESOURCE_BUSY` instead of `OUT_OF_RESOURCE`?

Reply:
Any return code has the same meaning, aka. no side effect and the upper layer needs to find another route to send the data.